src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/controller/PlanController.kt

```diff
@@ -66,4 +86,232 @@ class PlanController(
   ): PlanEntity {
     return planService.agreePlan(planUuid, agreement)
   }
+
+  @PostMapping("/{planUuid}/clone")
+  @Operation(
+    description = "Clones an existing sentence plan, creating a new plan with the same structure and details as the specified plan.",
+    tags = ["Integrations"],
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(responseCode = "200", description = "Plan cloned successfully"),
+      ApiResponse(
+        responseCode = "404",
+        description = "Plan not found",
+        content = arrayOf(Content(schema = Schema(implementation = ErrorResponse::class))),
+      ),
+      ApiResponse(
+        responseCode = "409",
+        description = "The plan could not be signed. See details in error message.",
```

Comment:
Why does this mention `signed`?

Reply:
I'm sure you can tell why.

Reply:
Resolved in https://github.com/ministryofjustice/hmpps-sentence-plan/pull/90/commits/12109d3313b957c52f46eb6221b529be88363d58